src/static/bootstrap/scss/bootstrap/_type.scss

```diff
@@ -8,6 +8,7 @@
 
 h1, h2, h3, h4, h5, h6,
 .h1, .h2, .h3, .h4, .h5, .h6 {
+  font-path: $font-path;
```

Comment:
Why did you put this in here? (I think there is not even a CSS property ``font-path`` and you can just omit this)